design-toolkit/components/src/components/badge/styles.ts

```diff
@@ -10,10 +10,11 @@
  * governing permissions and limitations under the License.
  */
 
+import { CRITICALITY_VARIANTS } from '@/constants/criticality-variants';
 import { tv } from '@/lib/utils';
 
 export const BadgeStylesDefaults = {
-  variant: 'info',
+  variant: CRITICALITY_VARIANTS.info,
```

Comment:
Not to be addressed in this PR, but we should update this from `variant` to `color` to be consistent with other components